mgr/manager.go

```diff
@@ -422,16 +526,17 @@ func (m *Manager) SetExpiresNext() {
 		log.Debugf("manager: %s expires at %s", m.Certs[i], cert.NotAfter)
 		if expires.After(cert.NotAfter) || expires.IsZero() {
 			expires = cert.NotAfter
+			nextCert = i
 		}
 	}
 
 	if expires.IsZero() {
 		log.Debug("manager: all certificates are set to renew")
-		metrics.ExpireNext.Set(0)
+		metrics.ExpireNext.WithLabelValues(m.Certs[nextCert].Spec.Path).Set(0)
```

Comment:
if `m.Certs` is empty then `nextCert` will be 0 so this will be out of bounds

Reply:
Good catch, thanks :)